fea-rs/src/compile/output.rs

```diff
@@ -38,9 +39,14 @@ pub struct Compilation {
 }
 
 impl Compilation {
-    /// Compile this output into a new binary font.
-    #[allow(clippy::result_unit_err)] //TODO: figure out error reporting
-    pub fn build_raw(&self, glyph_map: &GlyphMap, opts: Opts) -> Result<FontBuilder<'static>, ()> {
+    /// Generate all the final tables and add them to a builder.
+    ///
+    /// This builder can be used to get generate the final binary.
+    pub fn assemble(
+        &self,
+        glyph_map: &GlyphMap,
+        opts: Opts,
```

Comment:
Feels slightly odd that `compile` doesn't make me pass Opts but `assemble` does

Reply:
Just a consequence of the fact that options are currently only used during this last stage, because there is currently only one option. I do imagine this being used more widely in the future, but given we only have one user, I would prefer to break the API in the future based on known needs than to introduce API now that is anticipating future needs.

Reply:
Might be nicer if it followed the builder pattern consistently, e.g. if I want to give opts I'll call with_opts at an opportune time.